src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseProvider.cs

```diff
@@ -21,9 +23,9 @@ internal static class IntellisenseProvider
             new Intellisense.RecordNodeSuggestionHandler(),
         };
 
-        internal static IIntellisense GetIntellisense()
+        internal static IIntellisense GetIntellisense(PowerFxConfig powerFxConfig)
```

Comment:
>GetIntellisense [](http://example.com/codeflow?start=38&length=15)

Something here feels like we should have symmetry with how we specify which functions are available.

Reply:
We will. In the next PR I will do the same thing for built-in functions as I did for Enums. Make them non-static.